src/aiortc/codecs/vpx.py

```diff
@@ -436,6 +436,275 @@ def __update_config(self) -> None:
         self.cfg.rc_target_bitrate = self.__target_bitrate // 1000
         self.__update_config_needed = False
 
+class Vp9Decoder(Decoder):
+    def __init__(self) -> None:
+        self.codec = ffi.new("vpx_codec_ctx_t *")
+        _vpx_assert(
+            lib.vpx_codec_dec_init(self.codec, lib.vpx_codec_vp9_dx(), ffi.NULL, 0)
```

Comment:
Was this already exposed or did you have to do something to expose this API?

Reply:
Same as the `cx` function.